EasyVCR/Defaults.cs

```diff
@@ -14,60 +14,34 @@ internal static class Defaults
         /// </summary>
         internal static readonly Dictionary<string, object> ReplayHeaders = new Dictionary<string, object>
         {
-            { ViaRecordingHeaderKey, "true" },
+            { ViaRecordingHeaderKey, "true" }
         };
 
         /// <summary>
         ///     Default list of headers to censor in the cassettes.
         /// </summary>
         internal static List<string> CredentialHeadersToHide => new List<string>
         {
-            {
-                "authorization"
-            }
+            "authorization"
         };
 
         /// <summary>
         ///     Default list of parameters to censor in the cassettes.
         /// </summary>
         internal static List<string> CredentialParametersToHide => new List<string>
         {
-            {
-                "api_key"
-            },
-            {
-                "apiKey"
-            },
-            {
-                "key"
-            },
-            {
-                "api_token"
-            },
-            {
-                "apiToken"
-            },
-            {
-                "token"
-            },
-            {
-                "access_token"
-            },
-            {
-                "client_id"
-            },
-            {
-                "client_secret"
-            },
-            {
-                "password"
-            },
-            {
-                "secret"
-            },
-            {
-                "username"
-            }
+            "api_key",
```

Comment:
Thought: Ooo this is lovely, can we sort this?

Reply:
Resolved